openssl/plan.sh

```diff
@@ -1,7 +1,7 @@
 pkg_name=openssl
 _distname="$pkg_name"
 pkg_origin=core
-pkg_version=1.0.2r
+pkg_version=1.0.2t
```

Comment:
Please can you also update the windows plan.ps1 to the 1.0.2t version?

Reply:
Done